shared/tomcat/8.5/catalina.properties

```diff
@@ -108,7 +108,6 @@ shared.loader=
 tomcat.util.scan.StandardJarScanFilter.jarsToSkip=\
 azure.appservice.jar, \
 azure.appservice.easyauth.jar, \
-javax.json-*, \
```

Comment:
why is this no longer needed?

Reply:
https://github.com/Azure-App-Service/tomcat/blob/dev/shared/easyauth/javax.json-1.0.4.jar and https://github.com/Azure-App-Service/tomcat/blob/dev/shared/easyauth/javax.json-api-1.0.jar are not used anymore. @chunye: would be good to delete them.

Reply:
Use of javax.json* was removed by https://github.com/Azure-App-Service/tomcat/pull/22/files#diff-19a3ddf241387c5fa3007123bb153484

Reply:
These are deleted in this PR.